modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -37,150 +37,6 @@ $ env:MULE_HOME=C:\Downloads\mule-enterprise-standalone-4.1.5\
-+
-. After adding a new property, reinstall or restart the Windows Service.
-. Add the property and ensure that its index is greater than the greatest index in the `wrapper-additional.conf` file. Otherwise, your property can be overridden.
-+
-For example, if the `wrapper-additional.conf` has the property:
-+
-`wrapper.java.additional.n=-Dproperty=value`
-+
-When `n` is the greatest index, you should add your property in the `wrapper.conf` file incrementing the index:
-+
-`wrapper.java.additional.n+1=-DyourProperty=valueOfYourProperty`
-
-* Mule Enterprise Edition
-+
-. By default, you can add  20 extra properties after installing Mule as a Service.
-. Configure the number of extra properties you need, using the following command line argument
-when you install the service:
-+
-`-additionalJavaProperties=numberOfExtraProperties`
-
-== Starting Mule as a Linux/Unix Daemon
-
-To run Mule as a Linux-Unix Daemon, you need to install it first by running:
-
-[source,console]
-----
-$ $MULE_HOME/bin/mule install
-----
-
-Once installed, you can run Mule as a daemon:
-
-[source,console]
-----
-$ $MULE_HOME/bin/mule start
-----
-
-[NOTE]
-====
-You can also start Mule without executing the installation step. This will start the Mule in the background for the current user session.
-====
-
-This example starts Mule from a Unix console:
-
-[source,console,linenums]
-----
-$ $MULE_HOME/bin/mule start
-MULE_HOME is set to ~/Downloads/mule-enterprise-standalone-4.1.5
-MULE_BASE is set to ~/Downloads/mule-enterprise-standalone-4.1.5
-Starting Mule Enterprise Edition...
-Waiting for Mule Enterprise Edition.................
-running: PID:87318
-----
-
-== Deploying Mule Applications
-
-. Start Mule.
-. After Mule starts, you can deploy your Mule applications by moving your packaged JAR files to the `apps` directory in `$MULE_HOME`.
-
-== Stopping Mule
-
-Use the `stop` command to stop Mule.
-
-This example stops Mule from a Unix console:
-
-[source,console,linenums]
-----
-$ $MULE_HOME/bin/mule stop
-MULE_HOME is set to /Applications/mule-enterprise-standalone-4.1.5
-MULE_BASE is set to /Applications/mule-enterprise-standalone-4.1.5
-Stopping Mule Enterprise Edition...
-Stopped Mule Enterprise Edition.
-----
-
-== Removing the Mule Service or Daemon
-
-Use the `remove` command to remove the service or daemon from your system.
-
-This example removes the Mule daemon from a Unix console:
-
-[source,console,linenums]
-----
-$ $MULE_HOME/bin/mule remove
-MULE_HOME is set to /Applications/mule-enterprise-standalone-4.1.5
-MULE_BASE is set to /Applications/mule-enterprise-standalone-4.1.5
-Detected Mac OSX:
-Mule Enterprise Edition is not running.
-Removing Mule Enterprise Edition daemon...
-----
-
-== Enable Anypoint Monitoring
-
-Optionally, you can xref:monitoring::am-installing.adoc[install Anypoint Monitoring] for cloud-managed supported versions of on-premises runtimes so you can monitor applications running on that server.
-
 == See Also
```

Comment:
Do people not need to test the download? If they do, how will they do that if this content has moved?